web/modules/custom/ai_screening_project_track/src/Element/StaticSelect.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\ai_screening_project_track\Element;
+
+use Drupal\Core\Render\Element\Select;
+
+/**
+ * Static select element.
+ *
+ * @FormElement("ai_screening_static_select")
+ */
+class StaticSelect extends Select {
+  public const array STATIC_OPTIONS = [
+    'high' => 'High',
+    'average' => 'Average',
+    'low' => 'Low',
+    'irrelevant' => 'Irrelevant',
+  ];
```

Comment:
It may be better to use an `Enum` for these values to make it easier to compare values (in the future).